src/irmin/node.ml

```diff
@@ -307,6 +320,7 @@ module Make_generic_key
     (Node_key : Key.S with type hash = Hash.t) =
 struct
   module Core = Make_core (Hash) (Path) (Metadata) (Contents_key) (Node_key)
+  include Core
   include Of_core (Core)
```

Comment:
`Of_core` includes `Core`. No need to include it above

Reply:
as discussed offline, we need to include it because `Make_core` restricts its signature